### ISSUE TITLE
Question: Is possible to use ingress with Zalando Operator.

### ISSUE THREAD

**Issue by PGPAWAN** [zalando/postgres-operator#2507]
Just wanted to know is Zalando operator support ingress to connect with postgres pod?

**Using Production**: yes
**K8 version**: 1.23
**Operator Version**: postgres-operator:v1.9.0

**Comment by tarunmittal24** [comment]
yes @PGPAWAN, we can use the ingress in front of the postgress-pods, i am using it, and it is working perfectly.

**Comment by PGPAWAN** [comment]
@tarunmittal24  Thank for your response.. could please share share more details about ..how you are using if this possible.

**Comment by tarunmittal24** [comment]
I am using the kong-ingress deployed on the eks cluster and supports tcp traffic, load-balancer is NLB, as we need to use it for tcp traffic, and then just the ingress creation as normal we do. In kong, for tcp traffic routing, instead of Kind: Ingress we need to create the Kind: TCPIngress.

Route53--->NLB--->Kong Ingress Controller[with TCP Traffic supported] (by creating Kind: TCPIngress)----> SVC (postgress-service)---> PODS

**Comment by xieydd** [comment]
@tarunmittal24 Thank you very much for sharing the solution.

**Comment by xieydd** [comment]
@tarunmittal24 Have you enabled SNI in TCPIngress?
